src/session.ts

```diff
@@ -64,9 +64,14 @@ export class DebugSession implements IDebugger.ISession {
 
     if (client) {
       this._client.iopubMessage.connect(this._handleEvent, this);
+      this._clientChanged.emit(client as IClientSession);
```

Comment:
This PR would need to be rebased to take into account the changes from #231, which adds support for `Session.ISession`.

Reply:
resolved.